src/Simplex/Messaging/Crypto.hs

```diff
@@ -112,21 +110,21 @@ instance PrivateKey FullPrivateKey where
   mkPrivateKey = FullPrivateKey
 
 instance IsString FullPrivateKey where
-  fromString = parseString decodePrivKey
+  fromString = parseString (decode >=> decodePrivKey)
 
 instance IsString PublicKey where
-  fromString = parseString decodePubKey
+  fromString = parseString (decode >=> decodePubKey)
 
 parseString :: (ByteString -> Either String a) -> (String -> a)
 parseString parse = either error id . parse . B.pack
 
-instance ToField SafePrivateKey where toField = toField . serializePrivKey
+instance ToField SafePrivateKey where toField = toField . encodePrivKey
```

Comment:
to differentiate from `encode` / `decode` rename add prefix `binary_` to `encodePrivKey` / `encodePubKey`? - similar to parsers

Reply:
also maybe `import Data.ByteString.Base64 (decode, encode)`? had to go to ide for those